V6 project state

Clarence-F opened the pull request:
Added new Project State page

Commit:
Web API documentation and project configuration

Change-Id: Ie579d516d2eab0ed4e9b341c0da8487f2b5ab26a

Commit:
Add instructions for Windows to readme

Change-Id: I754295ef2a2c6f53b01748b85c7f3ab42aef67e5

Commit:
Fix path in readthedocs configuration

Change-Id: I52d6c8f38711cf3b1cd3d370e8b9ad35522b3cfd

sjmh3 commented on sphinx-src/web-api/index.rst:
```diff
@@ -30,3 +30,8 @@ Reference for controller web API endpoints.
    beacon
    channel
    reset
+   factory-reset
```
You've got some bits from another PR mixed in here.

sjmh3 commented on sphinx-src/web-api/project-state.rst:
```diff
@@ -0,0 +1,34 @@
+Project State
+#############
+.. note:: Expert only
```
```suggestion

.. note:: Expert only
```

sjmh3 commented on sphinx-src/web-api/project-state.rst:
```diff
@@ -0,0 +1,34 @@
+POST
+====
+
+Triggers changes to the Project State.
```
```suggestion
Requests a change to a project state.
```

sjmh3 reviewed (changes requested):
I'll need to look at the code to answer your todos.

Clarence-F committed:
Project State

Added new Project State page

Update project-state.rst

??? > TODO

Update index.rst

Clarence-F committed:
Fixes Implemented

Implemented Simon's comments
